typed/declaration/examples.ipldsch.json

```diff
@@ -4,53 +4,51 @@
 			"kind": "map",
 			"keyType": "String",
 			"valueType": "String",
-			"valueNullable": true
```

Comment:
I think I have this wrong, TypeMap uses "valueNullable" so I guess a "nullable" becomes "valueNullable" in that case